src/cms/templates/analytics/_broken_links_widget.html

```diff
@@ -14,7 +14,36 @@
             {% trans "Details" %}
         </a>
     </div>
-    <div class="p-3 w-full">
-        <!-- TODO -->
+    <div class="mx-8">
+    <table class="w-1/2 pl-8">
```

Comment:
```suggestion
    <table class="w-full max-w-xl">
```
I think this looks better on small screen sizes...